subprojects/docs/src/samples/userguide/dependencyManagement/managingTransitiveDependencies/importingDependencyConstraintsFromBOM/build.gradle

```diff
@@ -0,0 +1,20 @@
+apply plugin: 'java-library'
+
+repositories {
+    mavenCentral()
+}
+
+//START SNIPPET dependency-on-bom
+dependencies {
+    implementation 'org.springframework.boot:spring-boot-dependencies:1.5.8.RELEASE'
```

Comment:
Maybe comment on the fact that this is the dependency for the BOM.

Reply:
Good point. Makes it easier to get the example without reading the text. I added comments to this example and also the corresponding one in the release notes.